openvmm/openvmm_entry/src/cli_args.rs

```diff
@@ -93,6 +93,12 @@ pub struct Options {
     #[clap(long, requires("hv"))]
     pub get: bool,
 
+    /// The disk to use for the GET VMGS.
+    ///
+    /// If this is not provided, then a 4MB RAM disk will be used.
```

Comment:
tying back to my other comment wrt. the 4MB constant being an implementation defined constant - maybe we can just use `clap(long, default = "4MB")` to set this constant?